Cargo.toml

```diff
@@ -11,12 +11,12 @@ categories = ["game-engines", "rendering"]
 resolver = "2"
 
 [dependencies]
-bevy = { version = "0.9", default-features = false, features = ["render"] }
+bevy = { version = "0.10.0", default-features = false, features = ["bevy_render"] }

```

Comment:
@Fincap Does this also need feature `bevy_ui`? When I run cargo build I get errors because it depends on some UI features existing in the prelude.

Reply:
Good catch, thank you. I had only been testing with `cargo run`, but you're right `cargo build` fails unless `bevy_ui` is included, which is a bit surprising to me. I'm not overly familiar with cargo so I'm not certain why run works where build fails, I would be interested if anyone could provide insight on that.

The issue has been fixed.